recipes/mcstas/meta.yaml

```diff
@@ -0,0 +1,130 @@
+{% set version = "3.4.7" %}
+{% set sha_value = "e6114c0daa24b77dd69c84846da1505ed8c96b0bc46a91049450ccec21322266" %}
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: mcstas-suite
+  version: {{ version }}
+
+source:
+  url: https://github.com/McStasMcXtrace/McCode/archive/mcstas-{{ version }}.tar.gz
+  sha256: {{ sha_value }}
+  folder: src
+
+build:
+  number: 0
+
+outputs:
+
+  - name: mcstas-core
+    script: install-files-core.sh
```

Comment:
What's the motivation for the complex script over just copying over the source folder verbatim - i.e. 
```bash
cp -a "${SRCDATADIR}/." "${DESTDATADIR}/"
```